scaletest/templates/scaletest-runner/scripts/cleanup.sh

```diff
@@ -23,8 +23,8 @@ fi
 
 start_phase "Cleanup (${event})"
 coder exp scaletest cleanup \
-	--cleanup-job-timeout 15m \
-	--cleanup-timeout 2h |
+	--cleanup-job-timeout 2h \
+	--cleanup-timeout 5h |
```

Comment:
nit: I'm curious, should it be a little shorter than `workspace_pod_termination_grace_period_seconds`?

Reply:
We do it the other way, we add 1-2m to `workspace_pod_termination_grace_period_seconds` in the template instead. 😅

But we should probably parameterize these in the future.